presentation/src/main/java/me/androidbox/presentation/navigation/NavigationGraph.kt

```diff
@@ -83,7 +93,62 @@ fun NavigationGraph(
                 agendaScreenState = agendaScreenState,
                 agendaScreenEvent = { agendaScreenEvent ->
                     agendaViewModel.onAgendaScreenEvent(agendaScreenEvent)
-                })
+                },
+            onSelectedAgendaItem = {
+                /* TODO The item in the dropdown menu should be an enum or a sealed class that will determine which item was clicked */
+                navHostController.navigate(Screen.EventScreen.route)
+            })
+        }
+
+        /* Event Screen */
+        composable(
+            route = Screen.EventScreen.route
+        ) {
+            val eventViewModel: EventViewModel = hiltViewModel()
+            val eventScreenState by eventViewModel.eventScreenState.collectAsStateWithLifecycle()
+
+            EventScreen(
+                eventScreenState = eventScreenState,
+                eventScreenEvent = { eventScreenEvent ->
+                    eventViewModel.onEventScreenEvent(eventScreenEvent)
+                },
+                onEditTitleClicked = { title ->
+                    navHostController.navigate(route = "edit_screen/$title")
```

Comment:
If you are using the route many times, I recommend having a constant or something similar
Since having them written as strings all the time might lead to a typo and causing an issue

Reply:
@MKiperszmid Updated to use constants